src/schema/v2/sale/__tests__/index.test.js

```diff
@@ -229,6 +229,41 @@ describe("Sale type", () => {
         expect(data).toMatchSnapshot()
       })
     })
+
+    it("accepts the internalIDs argument", () => {
+      const query = `
+        {
+          sale(id: "foo-foo") {
+            saleArtworksConnection(internalIDs: ["sa-id-0", "sa-id-1"]) {
```

Comment:
The name `internalIDs` could be `ids`, but that might be confusing when the `id` refers to a base64-encoded global id. I wonder what name this argument deserves.

Reply:
Up to this point (so far as I'm aware) we don't use `internalID` anywhere else in the schema as an argument. Generally `node` is the only field that's queried with the GUID and the other `id` arguments are assumed to be `internalID`. 

@alloy correct me if I'm off on this. I _think_ this should be renamed to `id`.

Reply:
Indeed, we didn’t bother renaming arguments. So in other places in the schema where you will see `id` or `ids` arguments, it’s most likely referring to internal IDs or slugs. Having said that, perhaps it’s not a bad thing to start naming these more appropriately?

Reply:
I'm not strongly opinionated about this. Though we also seem to be using types (`String` and `ID`) to distinguish internal ids and node ids, so using the name `ids` here may not be that confusing.

Reply:
I think types help, but if I have a vote, I’d name internal ID for internal clarity within MP per Eloy’s comment above!

Reply:
> Though we seem to be using types (`String` and `ID`) as well to distinguish internal ids and node ids

That’s another thing and I actually do feel stronger about that. Identifier fields should imo be of type `ID`. The `ID` type is purely semantic, in that it represents an identifier value, and in reality is just a string.

Reply:
I would like to unblock a few other tasks and PRs that are dependents of this change. Are we okay with going with `internalIDs` and merging?

Reply:
Same, I'm not super opinionated, but I definitely can see how `internalID` is more consistent as an argument (over `id`) in V2, so I'm 👍 on adopting that!